static/app/components/replays/breadcrumbs/replayTimeline.tsx

```diff
@@ -26,30 +27,60 @@ const USER_ACTIONS = [
 ];
 
 function ReplayTimeline({}: Props) {
-  const {currentTime, duration, replay} = useReplayContext();
-  const crumbs = replay?.getRawCrumbs() || [];
+  const theme = useTheme();
+  const {currentHoverTime, currentTime, duration, replay} = useReplayContext();
+
+  if (!replay) {
+    return null;
+  }
+
+  const crumbs = replay.getRawCrumbs() || [];
+  const spans = replay.getRawSpans() || [];
   const transformedCrumbs = transformCrumbs(crumbs);
   const userCrumbs = transformedCrumbs.filter(crumb => USER_ACTIONS.includes(crumb.type));
 
+  const networkSpans = spans.filter(replay.isNotMemorySpan);
+
   return (
     <HorizontalMouseTracking>
-      <TimelineScubber />
       <Resizeable>
         {({width}) => (
           <Stacked>
             <MinorGridlines duration={duration || 0} width={width} />
             <MajorGridlines duration={duration || 0} width={width} />
-            <TimelinePosition currentTime={currentTime} duration={duration} />
-            <OffsetEvents crumbs={userCrumbs} width={width} />
+            <StackedUnderTimestamp>
+              <TimelinePosition
+                color={theme.purple300}
+                currentTime={currentTime}
+                duration={duration}
+              />
+              {currentHoverTime ? (
+                <TimelinePosition
+                  color={theme.purple200}
+                  currentTime={currentHoverTime}
+                  duration={duration}
+                />
+              ) : null}
+              <div>
```

Comment:
Is this needed or can we use a `Fragment`?

Reply:
a div or something real is needed. The parent of the `<div>` is a `<Stacked>` which puts everything overlapping on top of each other (same as position: absolute would).

We use the div here to say 'this is all one thing, the div should be `absolute`, but the children of the div are display:block"

I updated the code with an empty styled component and a comment to try and make the code more expressive: 
```
// Break out of a <Stacked> context, and go back to normal `display:block` mode.
const DisplayBlock = styled('div')``;
```